python/vyos/configtree.py

```diff
@@ -191,18 +194,27 @@
             else:
                 self.__set_add_value(self.__config, path_str, str(value).encode())
 
+        if self.__migration:
+            print(f"- op: set path: {path} value: {value} replace: {replace}")
```

Comment:
I think the bot has a point. If complete node paths and values are printed on stdout, that migration scripts that run at boot time will expose passwords and keys on the VGA screens and serial consoles for any curious datacenter technician to read. Or, in VMs, for any admin with access to VM consoles to read.

I believe we should either only log them in debug mode, or only to the log file, but not to stdout.

Reply:
They are not printed on stdout: since during migtation, configtree is called by the migration scripts, which are run by suprocess.*, with the output captured and logged,